src/ResourceManager/Sql/Commands.Sql/ServerUpgrade/Cmdlet/GetAzureSqlServerUpgrade.cs

```diff
@@ -21,6 +22,7 @@ namespace Microsoft.Azure.Commands.Sql.ServerUpgrade.Cmdlet
     /// <summary>
     /// Defines the Get-AzureRmSqlServerUpgrade cmdlet
     /// </summary>
+    [Obsolete("All Azure SQL DB Servers now have version 12.0 so there is nothing to upgrade.")]
```

Comment:
The Get commands actually still work, although they are useless. I will reinstate tests for these. The Start and Stop upgrade cmdlets can be made no-ops.